src/app/pages/homepage/homepage.component.html

```diff
@@ -1,78 +1,248 @@
+<head>
+  <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Fira+Sans:wght@600&display=swap" rel="stylesheet">S
+</head>
+
 <main>
```

Comment:
One of the housekeeping tasks next year would be to break down this homepage component into smaller components to increase the modularity of the code and keep the content of files small.